fusor-ember-cli/app/mixins/needs-deployment-mixin.js

```diff
@@ -6,6 +6,7 @@ export default Ember.Mixin.create({
   applicationController: Ember.inject.controller('application'),
 
   isStarted: Ember.computed.alias("deploymentController.isStarted"),
-  isNotStarted: Ember.computed.alias("deploymentController.isNotStarted")
+  isNotStarted: Ember.computed.alias("deploymentController.isNotStarted"),
 
+  disableCancel: Ember.computed.alias("isStarted")
```

Comment:
Curious why we do all of these aliases? More for my education than review of this PR :smile: